src/main/res/layout/dialog_share_proxy.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView
```

Comment:
Why a ScrollView here?

Reply:
general precaution but yeah it is unlikely that this screen could ever get so big, could remove it

Reply:
I removed it :+1: